AirCastingTests/SearchAndFollowTests/SessionForLocationDownloaderTests.swift

```diff
@@ -0,0 +1,87 @@
+// Created by Lunar on 02/08/2022.
+//
+import XCTest
+import Resolver
+@testable import AirCasting
+
+class SessionForLocationDownloaderTests: ACTestCase {
```

Comment:
I think it would be good to test the query string as well

Reply:
Agreed, but same as above. I will not focus on extending features, but rather on improvements.